L/Libxls/build_tarballs.jl

```diff
@@ -0,0 +1,33 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder
+name = "libxls"
+version = v"1.5.2"
+
+sources = [
+    "https://github.com/libxls/libxls/releases/download/v$version/libxls-$version.tar.gz" =>
+    "8d7e52d96ccc6c498e5de78c1988d9838d914eeeb94ac60208378340bd6e6aaa",
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir/libxls-*/
+ac_cv_func_malloc_0_nonnull=yes ac_cv_func_realloc_0_nonnull=yes ./configure --prefix=${prefix} --build=${MACHTYPE} --host=${target}
```

Comment:
Minor nitpick, I think that usually we pass these variables as arguments to configure, but the effect is the same, so it's fine

Reply:
```suggestion
./configure --prefix=${prefix} --build=${MACHTYPE} --host=${target} \
    ac_cv_func_malloc_0_nonnull=yes \
    ac_cv_func_realloc_0_nonnull=yes 
```